### ISSUE TITLE
Running the given codes gave a different output

### ISSUE THREAD

**Issue by HopeWu** [yangshun/front-end-interview-handbook#369]
### Running the given codes gave a different output.
```
function Parent() {
    this.name = 'Parent';
}

Parent.prototype.greet = function () {
    console.log('Hello from ' + this.name);
};

const child = Object.create(Parent.prototype);

child.cry = function () {
    console.log('waaaaaahhhh!');
};

child.cry();
// waaaaaahhhh!

/*
This actually prints "Hello from undefined"
*/
child.greet();
// hello from Parent

d.constructor;
// ƒ Parent() {
//   this.name = 'Parent';
// }

child.constructor.name;
// 'Parent'
```

The problem above is the following,
```js
/*
This actually prints "Hello from undefined"
*/
child.greet();
// hello from Parent
```

### The original page
https://www.frontendinterviewhandbook.com/javascript-questions/#example-of-prototypal-inheritance

**Comment by Devang-C** [comment]
Can you mention where the code is in this github repo. That would be helpful for us to solve the problem

**Comment by HopeWu** [comment]
Sure. Can you check pull request #371? The modification happens where the problem happens.

**Comment by gaurav19790** [comment]
is it still open I like to work on it

**Comment by yangshun** [comment]
Thanks for your suggestions, we've fixed it in d24b5d8
